lib/config-generator.js

```diff
@@ -632,7 +542,7 @@ class ConfigGenerator {
 
     buildWatchOptionsConfig() {
         const watchOptions = {
-            ignored: /node_modules/
+            ignored: 'node_modules'
```

Comment:
Can you explain this change?

Reply:
When I did that change using a RegExp was not allowed for that option anymore.

It seems that it was added back by https://github.com/webpack/webpack/pull/11498.

Reply:
I see it! We should probably change this back then?

Reply:
FYIU - this has been reverted